README.md

```diff
@@ -1,178 +1,229 @@
 
-* this function may be called multiple times to add multiple functions
-* `name` (string) - a name to identify the function.
-* `func` (function) - the test to benchmark.
+#### Params
+- **String** `name`: A name to identify the function.
+- **Function** `fun`: The test to benchmark.
+- **Object** `options`: The options object.
 
-**Specify inputs for functions**
+#### Return
+- **BenchTable** The `BenchTable` instance.
 
-```javascript
-Benchtable.prototype.addInput(name, input);
-```
+### `addInput(name, input)`
+Specify inputs for functions.
+This function may be called multiple times to add multiple inputs.
 
-* this function may be called multiple times to add multiple inputs
-* `name` (string) - a name to identify the input.
-* `input` (array) - the array containing arguments that will be passed to each benchmarked function.
-Therefore, the number of elements in the `input` array should match the number of arguments of each specified function (i.e. the array will be "unpacked" when invoking functions, they will not receive a single array argument).
+#### Params
+- **String** `name`: A name to identify the input.
+- **Array** `input`: The array containing arguments that will be passed to each benchmarked function. Therefore, the number of
+elements in the `input` array should match the number of arguments of
+each specified function (i.e. the array will be "unpacked" when
+invoking functions, they will not receive a single array argument).
 
-**Execute benchmarks**
+#### Return
+- **BenchTable** The `BenchTable` instance.
 
-```javascript
-Benchtable.prototype.run(config);
-```
+### Events
+
+#### `complete`
+Get results table.
 
-* `config` (object) -- [see benchmark.js API docs](http://benchmarkjs.com/docs#Suite_prototype_run)
+After all benchmarks are finished, the `complete` event is fired and the `cli-table` table instance with the results is
+available through the `BenchTable.prototype.table` property.
 
-**Get results table**
+The ASCII string serialization of the table is obtained by calling `.toString()` on that object.
 
-```javascript
-Benchtable.prototype.on('complete', function() {
-  this.table.toString();
+```js
+var suite = new BenchTable();
+...
+suite.on('complete', () => {
+  suite.table.toString();
 });
 ```
 
-* after all benchmarks are finished, the `complete` event is fired and the `cli-table` table instance with the results is available through the `Benchtable.prototype.table` property.
-The ascii string serialization of the table is obtained by calling `.toString()` on that object.
-
-## Example
+## Examples
 
 This is the Benchtable version of the example from the [Benchmark.js homepage](http://benchmarkjs.com).
-Also available in the [examples directory](https://github.com/izuzak/benchtable/examples/string_search.js).
+Also available in the [examples directory](/examples/string_search.js).
+
+### Default behavior
+
+```js
+// Import module
+var BenchTable = require('benchtable'); //require('benchtable');
```

Comment:
@izuzak It was there before (in the examples), and when running `blah -r` we [replace `../` with `benchtable`](https://github.com/izuzak/benchtable/commit/83d282d491bc886487ecb376f36b519aa9c408f7#diff-799839136677d9151042f5761071508cR17). :grin: